pkgs/development/libraries/webkitgtk/default.nix

```diff
@@ -93,6 +108,17 @@ stdenv.mkDerivation rec {
       src = ./fdo-backend-path.patch;
       wpebackend_fdo = libwpe-fdo;
     })
+  ] ++ [
+    # https://bugs.webkit.org/show_bug.cgi?id=232934
+    (fetchpatch {
+      url = "https://git.yoctoproject.org/poky/plain/meta/recipes-sato/webkit/webkitgtk/0001-Fix-build-without-opengl-or-es.patch";
+      sha256 = "sha256-9MUDNTda2yxo5knhBKu2PQXNASgTW2Z7H4F0aFt4b0Q=";
+    })
+    # https://bugs.webkit.org/show_bug.cgi?id=126433
+    ./fix-conflicting-types-on-darwin.patch
```

Comment:
```suggestion

    # Fix conflicting types on Darwin.
    # https://bugs.webkit.org/show_bug.cgi?id=126433
    # This was reverted and replaced by a different fix in
    # https://github.com/WebKit/WebKit/commit/ec643c6f6dd5c3d4702847cb79965029095ce024
    # but it does not work because of ???
    (fetchpatch {
      url = "https://github.com/WebKit/WebKit/commit/68822eb73f2cdd843dbe6bdd346e7268b279650b.patch";
      sha256 = "";
      excludes = [
        "Source/JavaScriptCore/ChangeLog"
      ];
    })
```

Reply:
Done, with an additional postFetch that changes `#ifdef __APPLE__` to `#if defined(__APPLE__)`.